1xx-enable-ssl-for-kafka-bridge.md

```diff
@@ -0,0 +1,137 @@
+
+## Proposal
+This proposal also adds new configurations to enable SSL server and specify locations of certificate and key files in PEM format so that client connections to HTTP Bridge can be encrypted. The new configurations will be added to the current `http` prefixed configurations:
+- http.ssl.enable
+- http.ssl.keystore.location
+- http.ssl.keystore.key.location
+- http.ssl.enabled.protocols
+- http.ssl.enabled.cipher.suites
+- http.admin.port
+
+When `http.ssl.enable` is set to true, HTTP Bridge server will be started with SSL enabled and the key and certificate loaded from the locations defined by `http.ssl.keystore.location` and `http.ssl.keystore.key.location` configurations. If the existing `http.port` configuration is not set, the server port will be set to `443` by default. This will allow users to make TLS encrypted connections from their clients for all the HTTP Bridge endpoints for client operations. 
+
+`http.ssl.enabled.protocols` can be configured with a comma separated list of enabled secure transport protocols that the server will accept from connecting clients. If not set, the server will use the same list of protocols that [Kafka use by default](https://kafka.apache.org/documentation/#brokerconfigs_ssl.enabled.protocols), which is `TLSv1.2,TLSv1.3`. 
+
+`http.ssl.enabled.cipher.suites` can be configured with a comma separated list of cipher suites that the server will support. If not set, the default list of cipher suites provided by the underlying JDK SSL/TLS engine will be used.
+
+This proposal also adds a separate listener for the endpoints that are used by internal clients such as
+- `/healthy`
+- `/ready`
+- `/metrics`
+
+The port for this listener will be set by the new configuration `http.admin.port` or 8081 by default.
+The rest of the endpoints will stay on the existing listener with the default port 8080 or 443 if SSL is enabled. 
+
+This brings a clear separation of internal and external use for the endpoints and makes it simpler to enable SSL for the listener that is used by external clients. It also brings a better performance and security because admin type of operations such as collecting metrics are isolated from traffic for Kafka operations.
+
+### HTTP Bridge managed by Strimzi operator
+
+When deploying KafkaBridge custom resources with the Strimzi operator, users can set these configurations via `spec.http` in their KafkaBridge CR. Users can enable SSL by setting the new field `spec.http.sslEnable` which is set to false by default. When it is set to true, users have to provide the certificate and key via Kubernetes secret that is referenced in the `certificateAndKey` field.
+
+Let's take a look at an example of KafkaBridge CR with the new fields to enable SSL:
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaBridge
+metadata:
+  name: my-bridge
+spec:
+  replicas: 3
+  bootstrapServers: <cluster_name>-cluster-kafka-bootstrap:9092
+  http:
+    port: 8443
+    adminPort: 8081 #NEW FIELD
```

Comment:
Should this be really configurable? I would probably argue against it. I don't think similar ports are configurable in other operands.